tutorials/reprojection.ipynb

```diff
@@ -937,13 +986,15 @@
    "id": "e16c4fbe",
    "metadata": {},
    "source": [
-    "#### WGS84 projection - lat/lon (equirectangular) projection (EPSG:4326)\n",
-    "This very common one displays meridians and parallels as equally spaced vertical and horizontal lines. Conversion between x, y coordinates on the map and earth locations are very simple, and this is a very popular one for general use. However, the significant distortions inherent to this method make it impractical for many scientific application. Let's see how our data looks in this crs by reprojecting it with [rasterio.warp.reproject](https://rasterio.readthedocs.io/en/latest/api/rasterio.warp.html#rasterio.warp.reproject) with following steps:\n",
+    "#### WGS 84 projection - lat/lon (equirectangular) projection (EPSG:4326) with rasterio's `reproject()`\n",
+    "[WGS 84](https://en.wikipedia.org/wiki/World_Geodetic_System) is commonly used in many geodetic applications, including the Global Positioning System (GPS). The WGS 84 projection displays meridians and parallels as equally spaced vertical and horizontal lines. Conversting between x and y coordinates on the map and earth locations is very simple. It is very popular for general use. However, the significant distortions inherent to this method makes it impractical for many scientific applications.\n",
+    "\n",
+    "\n",
+    "Let's see how our data looks in this crs by reprojecting it with [rasterio.warp.reproject](https://rasterio.readthedocs.io/en/latest/api/rasterio.warp.html#rasterio.warp.reproject) with following steps:\n",
     "\n",
-    "- Get shape, CRS, bounds and transform information from the source dataset.\n",
-    "- Define destination CRS.\n",
-    "- Calculate the shape, transform and coords of the destination.\n",
-    "- Reproject the source data to destination CRS and reconstruct the result."
+    "Follow these steps to reproject your data to WGS 84 with [rasterio.warp.reproject](https://rasterio.readthedocs.io/en/latest/api/rasterio.warp.html#rasterio.warp.reproject):\n",
+    "\n",
+    "**Step 1**: Get shape, CRS, bounds and transform information from the source dataset and define the destination CRS."
```

Comment:
For the 6 steps to work out, I had to slightly move some elements of your code around. I hope everything is still working correctly!